app/assets/stylesheets/sightEdit.scss

```diff
@@ -4,16 +4,14 @@
   text-align: center;
   margin: 0 auto;
   color: $blue-purple;
-
+  
   h1 {
     font-family: $header;
   }
```

Comment:
not sure if this PR was done before or after global styles but h1 styles should probably be covered by the h1 styles in application.scss